engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -23,7 +23,7 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
+  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson] = None, workflowOptions: Option[WdlJson] = None): Props = {
```

Comment:
@gauravs90 - @Horneth was suggesting something like [this](https://www.lucidchart.com/publicSegments/view/288aace2-3423-4364-a5f6-6eff306e754e/image.png), see the link he posted below. What I was arguing was that while I agreed with the drawing he put up there (ignoring the per-use worker or if it was a pool) that I thought that at the moment it'd be a premature optimization to do so and you'd just want to have that first layer VA. In a future where you want more workers behind it that VA address can simply become a router.

Does that make sense? If not, let's catch up on monday.